packages/good-design/src/core/modals/FVModal.tsx

```diff
@@ -87,4 +90,13 @@ function FVModal({ firstName, method, onClose = noop, ...props }: FVModalProps)
   );
 }
 
-export default FVModal;
+export const theme = {
+  baseStyle: withThemingTools(({ colorModeValue }: { colorModeValue: any }) => ({
+    justifyContent: "center",
```

Comment:
only fonts / colors / background / border colors  / shadow props are allowed in theme
please adjust all styles

check this as reference https://github.com/GoodDollar/GoodWeb3-Mono/pull/27/commits/3756062203e316edcd6e05a8bad616900930fcb6